### PR TITLE
Scrollbar

### LINKED ISSUE #4: Page scrolling

### PR DESCRIPTION
**Pull request by retrinh195**
Added the scrollbars to the GUI.

There are some issues with how the page is displayed after I added the scrollbars with the page not being in the center of the canvas.

I've tried a number of things, but the only thing I managed to add to attempt to fix the issue was make it so that the page stays in its intended spot on the canvas if the windows dimensions are changed (with exception to maximizing the window which will also be discussed later).

I have narrowed down the issue to being the thumbnail size being the issue. tkinter does not like when we try to manipulate by giving exact dimensions for the thumbail (e.g., 400 by 400) when resizing the image. I've also tried the solution of resizing the original dimensions twice and the same issues appeared. Basically setting the thumbnail of the image is the same as resizing the image twice, which tkinter does not like.

Maximizing the window also screws up how the page is displayed despite adding code that should've prevented this issue. It exhibits the same issue of the page being off-center when maximizing and restoring the window back to how it originally was.

The thumbnail and maximizing window issues could most likely be due to tkinter being the problem rather than how the code was written (but I could be wrong).

**TLDR**: Added scrollbar. Faced issues of page being displayed due to tkinter most likely being the problem after extensive testing.

Closes #4

### CODE REVIEW

**Comment by retrinh195** [comment]
Once we've merged this pull request, we should probably start working on the report and the presentation slides. I could probably add the last basic feature which would be the jump_to_page function since that was in the code when the pdf_gui.py file was added in the beginning.

If we are strapped on time, we could scrap the jump_to_page function if needed.

**Review by EricThang — APPROVED** [review]
honestly, since the scroll bar works and changing upon zooming in,  I think it's good enough.

**Review by imseano — APPROVED** [review]
